dotcom-rendering/src/components/FrontMostViewed.tsx

```diff
@@ -29,6 +30,7 @@ export const FrontMostViewed = ({
 	hasPageSkin,
 	isFront,
 	renderAds,
+	showKickers = false,
```

Comment:
I generally love this approach, where a `boolean` than `boolean | undefined`. If your `Props` make this a required property, as it currently does, you do not need to give this a default value.

I think you either want to make `showKickers` optional above or drop the default value here.